### ISSUE TITLE
No hardware resource monitoring in some cases

### ISSUE THREAD

**Issue by stocyr** [deepkit/deepkit-python-sdk#4]
When I launch an experiment from the Deepkit "Start" dialog using a `yml` config, I don't see any hardware resource monitoring metrics. I *do* see metrics when either launching a remote docker experiment from the Deepkit "Start" dialog or when launching locally by "Script run".

Setup:
- Windows 10
- Deepkit Release 2020.1.5
- Deepkit SDK 1.0.1

**Comment by marcj** [comment]
Yes, this is a current limitation of non-docker executions using the configuration file. The hardware monitoring of a execution directly on the host is just not implemented yet. This is rather a limitation directly in https://github.com/deepkit/deepkit and needs first some points to be clarified before it can be implemented (for example which GPU should be monitored? host-execution don't have any assigned)
